.github/workflows/release-wardend.yaml

```
@@ -0,0 +1,42 @@
+    strategy:
+      matrix:
+        release: ["release"]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
```

actionlint (1.7.4)

the runner of "actions/setup-go@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)